main.go

```
@@ -27,15 +31,71 @@ func loadEnv() error {
 
 func init() {
 	if err := loadEnv(); err != nil {
-		log.Fatal().Err(err).Msg("error loading environment variables")
+		log.Fatal().Err(err).Msg("Error loading environment variables")
 	}
 }
 
+func deleteUptimeChecksOneByOne(db *sql.DB, checks []UptimeCheck) (int64, error) {
```

golangci-lint

func `deleteUptimeChecksOneByOne` is unused

(unused)